build/cscope/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/bash
+#
+# {{{
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+# }}}
+#
+# Copyright 2020 Carsten Grzemba
+# Copyright 2021 OmniOS Community Edition (OmniOSce) Association.
+
+. ../../lib/functions.sh
+
+PROG=cscope
+VER=15.9
+PKG=ooce/developer/cscope
+SUMMARY="Cscope is a developer's tool for browsing source code"
+DESC="Cscope is a developer's tool for browsing source code. It has an impeccable"
+DESC+="Unix pedigree, having been originally developed at Bell Labs back in the"
+DESC+="for many years, and has been used to manage projects involving 20 million" 
```

Comment:
```suggestion
DESC+="Unix pedigree, having been originally developed at Bell Labs, "
DESC+="and has been used to manage projects involving 20 million " 
```
or find the missing bit of text :)

Reply:
found the missing line :)